contrib/automation_tests/orbit_tracks.py

```diff
@@ -16,10 +16,11 @@ def main(argv):
         ConnectToStadiaInstance(),
         FilterAndSelectFirstProcess(process_filter="hello_ggp"),
         Capture(),
-        # "sdma0" is not present on the DevKits, instead there is "vce0", so this tests for "sdma0 or vce0"
+        # We check for either "sdma0" or "vce0", to exist. Both are connected to the encoding of video frames.
         VerifyTracksExist(track_names=[
             "Scheduler", ("*sdma0*", "*vce0*"), "gfx", "All Threads", "hello_ggp_stand"
-        ]),
+        ],
+                          allow_duplicates=True),
```

Comment:
Formatting looks a bit off here...

Reply:
I changed the formatting to 
` ],  allow_duplicates=True),`
which I think looks nicer but yapf is changing it back.

Reply:
I thought we were using pylint also for the formatting. But anyways, if a formatter is generating this, I am fine with it.